src/Microsoft.ML.HalLearners/SymSgdClassificationTrainer.cs

```diff
@@ -33,48 +33,78 @@ namespace Microsoft.ML.Trainers.SymSgd
     using TPredictor = IPredictorWithFeatureWeights<float>;
 
     /// <include file='doc.xml' path='doc/members/member[@name="SymSGD"]/*' />
+    [BestFriend]
     public sealed class SymSgdClassificationTrainer : TrainerEstimatorBase<BinaryPredictionTransformer<TPredictor>, TPredictor>
     {
         internal const string LoadNameValue = "SymbolicSGD";
         internal const string UserNameValue = "Symbolic SGD (binary)";
         internal const string ShortName = "SymSGD";
 
-        public sealed class Arguments : LearnerInputBaseWithLabel
+        public sealed class Options : LearnerInputBaseWithLabel
         {
+            /// <summary>
+            /// Degree of lock-free parallelism. Determinism not guaranteed.
+            /// Multi-threading is not supported currently.
```

Comment:
>Multi-threading is not supported currently. [](start = 16, length = 43)

Does it mean that this argument is not used?

Reply:
good point! It doesn't seem like it is getting used. I know we switched the parallelism off in the  native code, due to MKL being sequential. I'll check if that is the case, and log an issue. 

---
In reply to: [248885364](https://github.com/dotnet/machinelearning/pull/2170#discussion_r248885364) [](ancestors = 248885364)